src/schema/config.ts

```diff
@@ -96,6 +95,6 @@ export interface ApiClientConfig {
     request_object_signing_alg: TokenSigningAlgorithm
     redirect_uri: string
     response_type: ResponseType
-    keys: JSONWebKey[]
+    keys: JWK[]
```

Comment:
This was a breaking change in terms of expected types [here](https://github.com/moneyhub/moneyhub-api-client/blob/master/src/index.ts#L49) 
open-id's client expects the `jose.JWK` type now